backend/development.ini.sample

```diff
@@ -95,9 +95,27 @@ basic_setup.session_secret = change_this_value_please!
 # app.enabled = contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,agenda,collaborative_document_edition,share_content,upload_permission,gallery
 
 ### Live Messages ###
-# control_uri for GRIP protocol reverse-proxy (PushPin)
+# control_uri for GRIP protocol reverse-proxy (PushPin) as a comma separated list
+# additional Pushpin instances can be provided
 # this is the base url used for publishing new events to reverse-proxy.
 ; live_messages.control_zmq_uri = tcp://localhost:5563
+
+# push_uri for Pushpin as a comma separated list
+# additional Pushpin instances can be provided
+# this is used to send events to Pushpin
+# it is automatically resolved through the control_zmq_uri if Pushpin is accessible on the same
+# ports as it's configuration
+; live_messages.push_zmq_uri =
+# live_messages.push_zmq_uri = tcp://localhost:5560
```

Comment:
add a trailing coma, so sysadmin naturally understand that this is a multivaluable parameter

Reply:
resolved in last commit